File: luxeballoondecor_website.client/src/app/models/CommunicationLogEntry.ts
import { CommunicationOutcomeEnum, ModeEnum } from "./Enumerators";

export interface CommunicationLogEntry {
  logID: number,
  personID: number,
  date: Date,
  mode: ModeEnum,
  content: string,
  multimediaContentID: number,
  outcome: CommunicationOutcomeEnum
}

File: LuxeBalloonDecor_website.Server/Models/CommunicationLogEntry.cs
using System.ComponentModel.DataAnnotations;

namespace LuxeBalloonDecor_website.Server.Models
{
    public class CommunicationLogEntry
    {
        [Key]
        public int logID { get; set; }
        public int personID { get; set; }
        public DateTime date { get; set; }
        public int mode { get; set; } // Mode Enum
        public string content { get; set; }
        public int? multimediaContentID { get; set; }
        public int outcome { get; set; } // Communication Outcome Enum
    }
}
